code/our-flowers.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kreis4 Flower Shop - Plants & Flowers</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+  <header> 
+    <nav class="hamburger">
+        <div></div>
+        <div></div>
+        <div></div>
+    </nav>
+
+    <menu>
+        <a href="./index.html">Home</a>
+        <a href="./our-flowers.html">Flowers</a>
+        <a href="./subscription.html">Subscription</a>
+        <a href="./faqs.html">FAQs</a>
+        <a href="./about.html">About</a>
+    </menu>
+
+    <div class="heading-desktop">
+      <h1>Welcome to Kreis4 Flowers</h1>
+      <h2>Fresh Flowers All Year Round</h2>
+  </div>
+  <div class="heading-mobile">
+      <h1>Kreis4 Flowers</h1>
+  </div>
+</header>
+
+    <h1>Our Flowers</h1>
```

Comment:
An `h1` is like the title of a book, there should only be one. Maybe this could be a lower level heading?